ppa/archive/tests/test_views.py

```diff
@@ -418,13 +418,37 @@ def test_digitizedwork_listview(self):
         assert not response.context['object_list'].count()
         self.assertContains(response, 'Invalid range')
 
+        # ajax request for search results
+        response = self.client.get(url, HTTP_X_REQUESTED_WITH='XMLHttpRequest')
+        assert response.status_code == 200
+        # should render the results list partial and single result partial
+        self.assertTemplateUsed('archive/snippets/results_list.html')
+        self.assertTemplateUsed('archive/snippest/search_result.html')
```

Comment:
Also check `assertTemplateNotUsed` on `archive/list_digitizedworks.html`

Reply:
I think I tried that but it's not a valid assertion?? I couldn't find an idiomatic way to represent that...but I'll take another look

Reply:
Does this not work? https://docs.djangoproject.com/en/1.11/topics/testing/tools/#django.test.SimpleTestCase.assertTemplateNotUsed

Reply:
huh, that definitely looks right. I think either I was trying `assertNotTemplateUsed`, or I did try that but Django was saying that the template _was_ used...I'll try it again.